hyperspy/misc/material.py

```diff
@@ -236,21 +237,220 @@ def density_of_mixture_of_pure_elements(weight_percent,
     Examples
     --------
     Calculate the density of modern bronze given its weight percent:
-    >>> hs.material.density_of_mixture_of_pure_elements(
-            (88, 12),("Cu", "Sn"))
+    >>> hs.material.density_of_mixture((88, 12),("Cu", "Sn"))
     8.6903187973131466
 
     """
     from hyperspy.signals import Signal
     elements = _elements_auto(weight_percent, elements)
     if isinstance(weight_percent[0], Signal):
         density = weight_percent[0]._deepcopy_with_new_data(
-            _density_of_mixture_of_pure_elements(
-                stack(weight_percent).data, elements, mean=mean))
+            _density_of_mixture(stack(weight_percent).data,
+                                elements, mean=mean))
         return density
     else:
-        return _density_of_mixture_of_pure_elements(weight_percent,
-                                                    elements, mean=mean)
+        return _density_of_mixture(weight_percent, elements, mean=mean)
+
+
+def mass_absorption_coefficient(element, energies):
+    """
+    Mass absorption coefficient (mu/rho) of a X-ray absorbed in a pure
+    material.
+
+    The mass absorption is retrieved from the database of Chantler2005
+
+    Parameters
+    ----------
+    element: str
+        The element symbol of the absorber, e.g. 'Al'.
+    energies: float or list of float or str or list of str
+        The energy or energies of the X-ray in keV, or the name of the X-rays,
+        e.g. 'Al_Ka'.
+
+    Return
+    ------
+    mass absorption coefficient(s) in cm^2/g
```

Comment:
Yeah I saw this but I thought mu was in units of barns/atom and rho was in units of g/cm^3 so mu/rho should be cm/g surely. Not necessarily needed to be understood for this pull to be merged maybe more of an issue for #962 .